### DISCUSSION TITLE
New version for F1 23?

### DISCUSSION THREAD

**Discussion by Ritchi3t** [masterplane8/F1-League-Management#23] [General]
Hello Masterplane8,

Are you planning to release a new version of this legendary sheet for F1 23?

**Comment by olewis14** [comment]
@Ritchi3t I am pretty sure the sheet will still work for F1 23. You can type in your own track name and abbreviation, so even though the drop down menu doesn't have the new tracks, you can enter them manually.  I don't actually own the game at the moment, so is there anything about the sheet that doesn't work for the new game right now?

**Reply by Ritchi3t** [reply]
Hello @masterplane8.
Yes everything is working fine wil be able to use it in the new game without an update, few tracks are missing indeed but easy to solve with the custom track list you already implemented. 
I would like to see to have separate stats for the sprint races so they are not included in the feature race stats

So just a small quality of life improvement is what i am looking for.
Thanks for your response! And thanks again for this sheet! Been using it for many years now, you are a legend!
